vertical-pod-autoscaler/pkg/updater/priority/update_priority_calculator.go

```diff
@@ -149,7 +151,23 @@ func (calc *UpdatePriorityCalculator) getUpdatePriority(pod *apiv1.Pod, recommen
 	// Sum of recommendations over all containers, per resource type.
 	totalRecommendedPerResource := make(map[apiv1.ResourceName]int64)
 
+	annotationValue, annotationExists := pod.GetAnnotations()[annotations.VpaObservedContainersLabel]
```

Comment:
observedContainers, hasObservedContainers?
It's not immediately clear what annotation you mean

Reply:
Done.